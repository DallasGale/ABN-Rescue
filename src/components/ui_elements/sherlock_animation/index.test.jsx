/* eslint-env jest */

import React from 'react'
import { render } from '@testing-library/react'

import SherlockAnimation from '.'

describe('<SherlockAnimation />', () => {
  it('renders without crashing', () => {
    const component = render(<SherlockAnimation />)
    expect(component).toMatchSnapshot()
  })
})
