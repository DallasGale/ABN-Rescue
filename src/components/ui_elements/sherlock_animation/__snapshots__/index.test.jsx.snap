// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SherlockAnimation /> renders without crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="styled__StyledContent-sc-1jtcv8m-0 fJwgMz"
      >
        <div
          class="background"
        >
          <div />
          <div />
        </div>
        <div
          class="face"
        >
          <div
            class="hand hand--left"
          >
            <div
              class="fingers"
            >
              <div
                class="finger"
              >
                <div
                  class="finger__tip"
                />
              </div>
              <div
                class="finger"
              >
                <div
                  class="finger__tip"
                />
              </div>
              <div
                class="finger"
              >
                <div
                  class="finger__tip"
                />
              </div>
              <div
                class="finger"
              >
                <div
                  class="finger__tip"
                />
              </div>
            </div>
          </div>
          <div
            class="eyes"
          >
            <div
              class="eye"
            >
              <div
                class="eye__ball"
              >
                <div
                  class="eye__pupil"
                />
              </div>
              <div
                class="eye__lid"
              />
            </div>
            <div
              class="eye"
            >
              <div
                class="eye__ball"
              >
                <div
                  class="eye__pupil"
                />
              </div>
              <div
                class="eye__lid"
              />
            </div>
          </div>
          <div
            class="hand hand--right"
          >
            <div
              class="fingers"
            >
              <div
                class="finger"
              >
                <div
                  class="finger__tip"
                />
              </div>
              <div
                class="finger"
              >
                <div
                  class="finger__tip"
                />
              </div>
              <div
                class="finger"
              >
                <div
                  class="finger__tip"
                />
              </div>
              <div
                class="finger"
              >
                <div
                  class="finger__tip"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="styled__StyledContent-sc-1jtcv8m-0 fJwgMz"
    >
      <div
        class="background"
      >
        <div />
        <div />
      </div>
      <div
        class="face"
      >
        <div
          class="hand hand--left"
        >
          <div
            class="fingers"
          >
            <div
              class="finger"
            >
              <div
                class="finger__tip"
              />
            </div>
            <div
              class="finger"
            >
              <div
                class="finger__tip"
              />
            </div>
            <div
              class="finger"
            >
              <div
                class="finger__tip"
              />
            </div>
            <div
              class="finger"
            >
              <div
                class="finger__tip"
              />
            </div>
          </div>
        </div>
        <div
          class="eyes"
        >
          <div
            class="eye"
          >
            <div
              class="eye__ball"
            >
              <div
                class="eye__pupil"
              />
            </div>
            <div
              class="eye__lid"
            />
          </div>
          <div
            class="eye"
          >
            <div
              class="eye__ball"
            >
              <div
                class="eye__pupil"
              />
            </div>
            <div
              class="eye__lid"
            />
          </div>
        </div>
        <div
          class="hand hand--right"
        >
          <div
            class="fingers"
          >
            <div
              class="finger"
            >
              <div
                class="finger__tip"
              />
            </div>
            <div
              class="finger"
            >
              <div
                class="finger__tip"
              />
            </div>
            <div
              class="finger"
            >
              <div
                class="finger__tip"
              />
            </div>
            <div
              class="finger"
            >
              <div
                class="finger__tip"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
